library/globals.inc.php

```diff
@@ -175,9 +175,9 @@ function gblTimeZones()
             xl('Pick a general theme (need to logout/login after change this setting).')
         ),
 
-        'font-family' => [
```

Comment:
Yep, ≥ 5.4. Agree with @bradymiller that you should avoid use of PHP 5.3 as it reached end of life way back in 2014 (http://php.net/supported-versions.php)